File: dotnet/RelevantCodes.ExtentReports/RelevantCodes.ExtentReports/Model/ExceptionInfo.cs
using System;
using RelevantCodes.ExtentReports.Model;

namespace RelevantCodes.ExtentReports.Model
{
    public class ExceptionInfo
    {
        public Exception Exception { get; private set; }

        public string ExceptionName { get; private set; }

        public string StackTrace { get; private set; }

        public Test Test { get; private set; }

        public ExceptionInfo(Test test, Exception exception)
        {
            this.Test = test;
            this.Exception = exception;

            ExceptionName = exception.Message;
            StackTrace = exception.StackTrace;
        }
    }
}

File: src/main/java/com/relevantcodes/extentreports/model/ExceptionInfo.java
package com.relevantcodes.extentreports.model;

import java.io.Serializable;

public class ExceptionInfo implements Serializable {

	private static final long serialVersionUID = -7676305793226138602L;

	private String exceptionName;
    private String stackTrace;
    private Test test;

    public String getExceptionName() {
        return exceptionName;
    }

    public String getStackTrace() {
        return stackTrace;
    }

    public Test getTest() {
        return test;
    }

    public void setTest(Test test) {
        this.test = test;
    }

    public void setExceptionName(String exceptionName) {
        this.exceptionName = exceptionName;
    }

    public void setStackTrace(String stackTrace) {
        this.stackTrace = stackTrace;
    }
}
